foodloop-db-files/01_schema.sql

```diff
@@ -49,7 +52,8 @@ CREATE TABLE foodloop.peopleToFoodlist (
 CREATE TABLE foodloop.foodlistToFood (
     foodlistID INT,
     foodID INT,
-    PRIMARY KEY (foodlistID, foodID),
+    foodIndex INT,
+    PRIMARY KEY (foodlistID, foodIndex),
```

Comment:
should revert, foodIndex is meant for the positioning of the food in the foodlist, so it doesn't make sense for it to be part of the primary key

Reply:
but since multiple of the same `foodID` can exist within the same foodlist, shouldn't `foodIndex` be part of the primary key so that each entry is uniquely identifiable by `(foodlistID, foodIndex)`?

Reply:
Both makes sense here. depends on whether we are allowing the same food to appear twice in the foodlist. if so we need another pk, mayb just a `id` field would suffice. If we are not allowing then this is fine. Don't need to think so much about it, but good catch. This can be knowledge to use when people ask questions during the demo.